src/urbit/app/chess.hoon

```diff
@@ -680,6 +691,17 @@
           %chess-game  !>(game)
```

Comment:
Mark here would be renamed to `%chess-game-active`, as well as on lines:
- `156`
- `622`
- `768`
- `806`